fabulaws/library/wsgiautoscale/api.py

```diff
@@ -1299,6 +1300,8 @@ def mount_encrypted(drive_letter='f'):
         print '*** WARNING: While immediately restarting a cache server works as expected, '\
               'stopping and later restarting does not. For more information, see docs/servers/maintenance.rst ***'
     if 'web' in _current_roles():
+        # supervisor may have failed to start during initial boot
```

Comment:
Does this have to be in both `if` statements? Are there other roles for which it's unsafe to (re)start `supervisor`?